api/configs/middleware/__init__.py

```diff
@@ -8,20 +8,20 @@
 from configs.middleware.storage.azure_blob_storage_config import AzureBlobStorageConfig
 from configs.middleware.storage.google_cloud_storage_config import GoogleCloudStorageConfig
 from configs.middleware.storage.tencent_cos_storage_config import TencentCloudCOSStorageConfig
-from configs.middleware.vdb.chroma_configs import ChromaConfigs
-from configs.middleware.vdb.milvus_configs import MilvusConfigs
-from configs.middleware.vdb.opensearch_configs import OpenSearchConfigs
-from configs.middleware.vdb.oracle_configs import OracleConfigs
-from configs.middleware.vdb.pgvector_configs import PGVectorConfigs
-from configs.middleware.vdb.pgvectors_configs import PGVectoRSConfigs
-from configs.middleware.vdb.qdrant_configs import QdrantConfigs
-from configs.middleware.vdb.relyt_configs import RelytConfigs
-from configs.middleware.vdb.tencent_vector_configs import TencentVectorDBConfigs
-from configs.middleware.vdb.tidb_vector_configs import TiDBVectorConfigs
-from configs.middleware.vdb.weaviate_configs import WeaviateConfigs
-
-
-class StorageConfigs(BaseModel):
+from configs.middleware.vdb.chroma_configs import ChromaConfig
```

Comment:
The Python filename seems like it can also remove the 's'.

Reply:
Thanks for the reminder. Fixed.

Reply:
Thanks for the reminder. Fixed.